AssignmentJenkins/FreeStyle-DSL-Script/FreeStyle_DSL_Script.groovy

```diff
@@ -0,0 +1,12 @@
+job('DSL-Tutorial-1-Test') {
+    scm {
+        git('https://github.com/leereilly/hello-world-java.git')
+    }
+    triggers {
+        scm('H/1 * * * *')
```

Comment:
what is the timing for triggering the job?

Reply:
It will poll code from repository every minute.